src/TConsole.cpp

```diff
@@ -2253,6 +2253,25 @@ TLabel* TConsole::createLabel(const QString& name, int x, int y, int width, int
     }
 }
 
+std::pair<bool, QString> TConsole::deleteLabel(const QString& name)
```

Comment:
This should also raise an event - so that in case someone makes a Geyser label and then uses `deleteLabel(name)` on it and not the Geyser method, Geyser could still keep its state intact.

Reply:
:thinking: I guess so - I hadn't thought of that aspect, I was thinking more that whoever is maintaining Geyser these days (who *is* that BTW?) will want to provide a revised `Geyser.label:close()` that perhaps takes an optional `bool` argument "destroy" which defaults to `false` - I am looking to do that with the *deliberately undocumented* `closeUserWindow()` so that those in the know *can* destroy sub-consoles / user windows / buffers - which they can't do right now!

Reply:
We maintain Geyser so that's on us to update :)

Reply:
Okay now there is a `"sysLabelDeletion"` event - providing just the label name as an parameter/argument...